templates/common/on-prem/files/NetworkManager-resolv-prepender-dispatch.yaml

```
@@ -0,0 +1,46 @@
+mode: 0755
+path: "/etc/NetworkManager/dispatcher.d/30-resolv-prepender"
+contents:
+  inline: |
+    #!/bin/bash
+    set -eo pipefail
+    IFACE=$1
+    STATUS=$2
+
+    >&2 echo "NM resolv-prepender dispatcher script triggered by ${1} ${2}."
```

```suggestion
    >&2 echo "NM resolv-prepender dispatcher script triggered by ${IFACE} ${STATUS}."
```
or remove line 7 & 8